ResourceGenerator/Module.Template.psm1

```diff
@@ -90,11 +90,11 @@ function Get-TargetResource
         ResourceGenerator#>
         if ($null -eq $getValue)
         {
-            Write-Verbose -Message "Nothing with <FilterScript> {$<FilterScript>} was found"
+            Write-Verbose -Message 'Nothing with <FilterScript> {$<FilterScript>} was found'
```

Comment:
This change will break the string output of the variable after the replacement. There need to be proper quotes for this string.

Reply:
Done